syntax/list.php

```diff
@@ -6,27 +6,36 @@
  * @author  Andreas Gohr, Michael Große <dokuwiki@cosmocode.de>
  */
 
+use dokuwiki\plugin\struct\meta\AggregationList;
+use dokuwiki\plugin\struct\meta\ConfigParser;
+use dokuwiki\plugin\struct\meta\SearchConfig;
+use dokuwiki\plugin\struct\meta\StructException;
+
 // must be run within Dokuwiki
 if (!defined('DOKU_INC')) die();
 
 class syntax_plugin_struct_list extends DokuWiki_Syntax_Plugin {
+
+    /** @var string which class to use for output */
+    protected $tableclass = AggregationList::class;
```

Comment:
OTOH it seems this class and the cloud syntax class duplicate a bit of code from the table and lookup classes. So maybe it's worth to let them inherit and only overwrite a few things?